satpy/tests/test_multiscene.py

```diff
@@ -552,49 +572,217 @@ def test_crop(self):
         self.assertTupleEqual(new_scn1['4'].shape, (92, 357))
 
 
-class TestBlendFuncs(unittest.TestCase):
+class TestBlendFuncs():
     """Test individual functions used for blending."""
 
-    def setUp(self):
-        """Set up test data."""
-        from datetime import datetime
+    @pytest.fixture(autouse=True)
+    def _get_line_column(self):
+        self._line = 2
+        self._column = 3
```

Comment:
How about a `setup_method` method that assigns these to `self`? I'm a little scared of using an autouse fixture that may or may not apply to all tests run in satpy.

Reply:
Done